CITATION.cff

```diff
@@ -0,0 +1,50 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: FuzzBench
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Jonathan
+    family-names: Metzman
+    email: metzman@google.com
+    affiliation: Google LLC
+    orcid: 'https://orcid.org/0000-0002-7042-0444'
+  - given-names: Laszlo
+    family-names: Szekeres
+    email: lszekeres@google.com
+    affiliation: Google LLC
+    orcid: 'https://orcid.org/0000-0002-9808-6484'
+  - given-names: Laurent
+    family-names: Simon
+    email: laurentsimon@google.com
```

Comment:
Is there a preferred alternative email?
Or is it preferred to omit emails from this repo?

Reply:
Probably just omit.